ruby/example_code/rds/list_security_groups.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[rds.ruby.listSecurityGroups]
+require "aws-sdk-rds"  # v2: require 'aws-sdk'
+
+# List all Amazon Relational Database Service (Amazon RDS) security groups.
+#
+# @param rds_resource [Aws::RDS::Resource] An SDK for Ruby Amazon RDS resource
+# @return security_groups [List, nil] A list of security groups or nil if error
```

Comment:
@return security_groups [List, nil] A list of security groups, or nil if error.

Reply:
Done.